packages/core/monaco-editor/src/composables/useMonacoEditor.ts

```diff
@@ -2,17 +2,10 @@ import { onActivated, onBeforeUnmount, onMounted, reactive, toValue, watch } fro
 import { DEFAULT_MONACO_OPTIONS } from '../constants'
 import { unrefElement, useDebounceFn } from '@vueuse/core'
 
-// TODO: this will be replaced in the future so we only import the needed modules and features
```

Comment:
Whether or not the Vite plugin is introduced, the runtime source code remains unchanged.